main.go

```diff
@@ -20,22 +20,18 @@ import (
 	"flag"
 	"os"
 
-	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
-	// to ensure that exec-entrypoint and run can make use of them.
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
-
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
```

Comment:
Wondering why this moved up here (i.e. away from the other `github.com` imports). I have seen `go fmt` move things around, but it seems fine with this.

Reply:
Oh yeah, that my IDE running `gosimports` (https://github.com/rinchsan/gosimports) to automatically organize imports into groups and sort them.